app/assets/stylesheets/admin/components/_inline-label.scss

```diff
@@ -0,0 +1,5 @@
+.label--inline {
```

Comment:
This file should just be called `label` now as it should be named after the block